packages/api-explorer/src/JsonForm/index.jsx

```diff
@@ -21,6 +22,9 @@ function configureJSONEditor(intl, setFormSubmissionListener) {
     JSONEditor.defaults.editors[key] = getCustomEditor(key);
   });
 
+  JSONEditor.defaults.languages.it = it
```

Comment:
Don't we need to set the same for `en` too?

Reply:
by default it is `en`

Reply:
ok